android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/provider/DevicePowerConfigProvider.java

```diff
@@ -0,0 +1,25 @@
+package com.solana.mobilewalletadapter.walletlib.provider;
+
+import android.content.Context;
+import android.os.PowerManager;
+
+import androidx.annotation.Nullable;
+
+public class DevicePowerConfigProvider implements PowerConfigProvider {
```

Comment:
Since this is a default implementation, I'd move it into the same package which consumes it, and make it a package-visibility class